uaclient/apt.py

```diff
@@ -467,7 +467,9 @@ def clean_apt_files(*, _entitlements=None):
     from uaclient.entitlements.repo import RepoEntitlement
 
     if _entitlements is None:
-        from uaclient import entitlements as _entitlements
+        from uaclient import entitlements as __entitlements
+
+        _entitlements = __entitlements
```

Comment:
I mean, the tests for this are just plain wrong and I hate it, but I understand this has nothing to do with this specific PR. We will rewrite this file anyway.